py/server/deephaven/parquet.py

```diff
@@ -76,30 +79,55 @@ def _build_parquet_instructions(
     if max_dictionary_size is not None:
         builder.setMaximumDictionarySize(max_dictionary_size)
 
-    builder.setIsLegacyParquet(is_legacy_parquet)
+    if is_legacy_parquet:
+        builder.setIsLegacyParquet(is_legacy_parquet)
 
     if target_page_size is not None:
         builder.setTargetPageSize(target_page_size)
 
-    builder.setIsRefreshing(is_refreshing)
+    if is_refreshing:
+        builder.setIsRefreshing(is_refreshing)
 
     return builder.build()
 
+class ParquetType(Enum):
+    """ The parquet file layout type. """
+
+    SINGLE = 1
```

Comment:
SINGLE_FILE

Reply:
So, I do think it's a _bit_ redundant to explicitly call out "FILE"; the equivalent for the others would be `FLAT_PARTITIONED_DIRECTORY`, etc. The docs do mention "file" or "directory" as appropriate.

I do think these types also cleanly map into what we might expect w/ s3. So, `s3://my-bucket/foo.parquet` should work with `SINGLE`. And I do think the existing documentation probably still works in the s3 case - the big different from the parquet side is that `path (str): the file or directory to examine` needs to be expanded to include other URIs.

I'm not wholly against changing, just not sure it's helpful.

Reply:
I think SINGLE_FILE is more clear.

Reply:
That s3 object is a parquet file in every meaningful away.

Reply:
Look at language Amazon uses:
https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-etl-format-parquet-home.html

Reply:
You've convinced me.